---
title: About
date: "2020-10-22T00:00:00.000Z"
description: "About page."
---

**EPI2ME Labs** maintains a collection of example bioinformatics notebooks for
running analysis in your web browser. Our
[GitHub](https://github.com/epi2me-labs/tutorials) repository contains a
growing collection of notebooks on a range of topics from basic quality control
to genome assembly. These are free and open to use by anyone.

EPI2ME Labs extends the JupyterLab notebook framework with a pre-configured
analysis server. The notebooks contain Python code needed to run data analysis,
however zero configuration of the code is required. The notebooks can be used
both by **students** and **researchers** new to nanopore sequence data
analysis, and more experienced bioinformaticians who are comfortable working
with Python.

The EPI2ME Labs server runs in a [Docker
container](https://www.docker.com/resources/what-container) to provide an
isolated and resettable environment.


## Getting started

EPI2ME Labs notebooks are supported on the GridION Mk1 sequencing platform as
well as any Windows 10, macOS, or Linux desktop. All components are also
released open-source in our [GitHub](https://github.com/epi2me-labs)
repository.

The [Quick Start guide](/quickstart) will help you to get started running a
notebook server and web-based analyses.

Our [Index of Notebooks](/nbindex) page to see a complete listing with
descriptions.

The [Downloads](/downloads) page contains a list of all available resources to
get you started.

To learn more about the Jupyter project, see
[jupyter.org](https://www.jupyter.org/).
